CHANGELOG.rst

```diff
@@ -158,6 +158,17 @@ New Features
   from our private TUF repository. Please note any third party dependencies will
   still be downloaded from PyPI with no additional security validation.
 
+Upgrade Notes
```

Comment:
We should update the doc: https://github.com/DataDog/datadog-agent/blob/master/docs/agent/upgrade.md

Reply:
I didn't update this page of the docs because this doesn't affect upgrades from Agent 5: Agent 5 only supported HTTP connections to proxies, and the import script imports the Agent5 settings correctly (see https://github.com/DataDog/datadog-agent/blob/6.3.3/pkg/legacy/converter.go#L232 + agent 5 didn't support a scheme in `proxy_host`)